packages/reqresp/src/protocols/LightClientOptimisticUpdate.ts

```diff
@@ -15,5 +15,17 @@ export const LightClientOptimisticUpdate: ProtocolDefinitionGenerator<null, alta
     requestType: () => null,
     responseType: () => ssz.altair.LightClientOptimisticUpdate,
     contextBytes: getContextBytesLightclient((update) => modules.config.getForkName(update.signatureSlot), modules),
+    inboundRateLimits: {
+      /**
+       * Optimistic updates can't be passed more frequently than once per slot.
+       * So for one peer we allow more relaxed double rate limit.
+       *
+       * 12 seconds is chosen to be fair and relates to slot but can be updated in future.
+       *
+       * For total we multiply with `10` to have lower peer count on light client.
+       */
+      byPeer: {quota: 2, quotaTime: seconds(12)},
+      total: {quota: 20, quotaTime: seconds(10)},
```

Comment:
Setting this rate limit values here is fundamentally wrong since you are "forcing" the rate limit values of a full-node to every consumer of this library which includes full nodes and light client consumers.

Reply:
It's not **forcing** the rate limits rather setting **default** values. Any consumer of `reqresp` package can change these independently at the time of protocol registration.

Reply:
Right, tho there's a debate if the default should be no value at all (completely agnostic), or we apply the default that's convenient to our specific usecase

Reply:
I believe our use case should be similar to lot others, so setting default values is much convenient. If one had a unique use can he still can register the protocol with different limits, there is no restriction on it.